tests/php/src/Cli/Import/ImportWxrFile.php

```diff
@@ -40,13 +47,69 @@ final class ImportWxrFile implements ImportStep {
 	 */
 	private $wxr_file;
 
+	/**
+	 * Whether the WXR file is a temporary file and needs cleanup.
+	 *
+	 * @var bool
+	 */
+	private $wxr_is_temporary_file = false;
+
 	/**
 	 * ImportWxrFile constructor.
 	 *
 	 * @param string $wxr_file File path to the WXR file to import.
 	 */
 	public function __construct( $wxr_file ) {
 		$this->wxr_file = $wxr_file;
+
+		WP_CLI::warning( $this->wxr_file );
```

Comment:
Why `warning`? Is this debug code?

```suggestion
		WP_CLI::log( $this->wxr_file );
```